packages/pytest-simcore/src/pytest_simcore/schemas.py

```diff
@@ -106,6 +106,11 @@ def _run_diff(schema_lhs: dict, schema_rhs: dict) -> subprocess.CompletedProcess
         schema_rhs_path = tmpdir / "schema_rhs.json"
         schema_rhs_path.write_text(json.dumps(schema_rhs, indent=1))
 
+        # NOTE: When debugging the differences, as of now both schemas come from
+        # pydantic model, now it is possible to visually compare the difference. To do so,
+        # just dereference the current pydantic schema. Example can be seen here:
```

Comment:
ok so once your version is correct. these tests might disappear. they are here only because we still used the old version of the openapi.

Reply:
I would still keep them, they are very useful if there is some change in the Pydantic behavior after the upgrade. For example https://github.com/ITISFoundation/osparc-simcore/issues/3905 after the version upgrade there was a change in the json schema which would be caught by this test